.github/workflows/codeql.yml

```diff
@@ -0,0 +1,68 @@
+name: "CodeQL code scanning"
+
+on:
+  push:
+    branches: [ "master", "beta", "rc" ]
+  pull_request:
+    branches: [ "master", "beta", "rc" ]
+  schedule:
+    - cron: '35 22 * * 3'
```

Comment:
Why 22:35 on Wednesdays?

Reply:
I think that's randomly generated to avoid batching on server